exercises/parallel-letter-frequency/README.md

```diff
@@ -33,11 +33,11 @@ If you are using rustup.rs:
 rustup run nightly cargo bench
 ```
 
-- [Benchmark tests](https://doc.rust-lang.org/book/benchmark-tests.html)
+- [Benchmark tests](https://doc.rust-lang.org/1.1.0/book/benchmark-tests.html)
 
 Learn more about nightly Rust:
 
-- [Nightly Rust](https://doc.rust-lang.org/book/nightly-rust.html)
+- [Nightly Rust](https://doc.rust-lang.org/nightly/)
```

Comment:
This link doesn't point to documentation _about_ nightly Rust, but to the documentation overview page for the day's nightly build.

Reply:
Ah, my mistake. Would you agree with [https://doc.rust-lang.org/book/first-edition/release-channels.html](https://doc.rust-lang.org/book/first-edition/release-channels.html) as a good place to direct users? I cannot see anything more specific regarding nightly releases.

Reply:
Yes, with the caveat that once the second-edition book's "nightly" appendix is written, that would be better yet. As of right now, your first-edition link is the best option available.